src/components/VolunteerAutomation/VerifyPhoneNo.js

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import { Typography, Container, Box, TextField } from "@mui/material";
+import "./styles.scss";
+
+function VerifyPhoneNo({ setDisable, setContact, contact }) {
+  let number = "";
+  const handleChange = async (event) => {
+    number += event.target.value;
+    if (number.length === 10) {
+      setDisable(false);
+    }
+    if (number.length > 10) {
+      setDisable(true);
+    }
+    if (number.length < 10) {
+      setDisable(true);
+    }
+    setContact(event.target.value);
+  };
+
+  return (
+    <Container sx={{ mt: 5 }} maxWidth="sm">
+      <Typography variant="h6" gutterBottom>
+        Please let us verify your phone number
+      </Typography>
+      <Typography variant="body1" gutterBottom>
+        We will only use your number to contact you for volunteer related
+        matters. We never share it to any third party.
+      </Typography>
+      <Box sx={{ mt: 4 }}>
+        <TextField
+          label="Phone Number"
+          type="number"
```

Comment:
From the earlier discussion about why I'd recommend using `tel` instead of `number` here https://github.com/navgurukul/bhanwari-devi/pull/229#discussion_r791803501:
> Using type='tel' gives the browser (and search engines) better semantic information about the form element. If you use type = 'number', then you may have up/down arrows on the right of the form field, which could be confusing to the user as you don't usually enter telephone numbers that way

```suggestion
          type="tel"
```